packages/live-preview-sdk/src/csm/types.ts

```diff
@@ -0,0 +1,30 @@
+export type Source = {
+  field: number;
+  locale: number;
+} & ({ entry: number } | { asset: number });
+
+export interface EntitySource {
+  space: number;
+  environment: number;
+  id: string;
+}
+
+type Mappings = Record<string, { source: Source }>;
+
+interface IContentSourceMaps {
```

Comment:
Why do we have the `I` prefix here? 🤔

Reply:
Ahh, this was in the same file as the ContentSourceMaps class, so I needed to rename it (I for interface). But since I moved it to its own types file we can remove the I